### ISSUE TITLE
about the result visualization

### ISSUE THREAD

**Issue by hbsycym** [JihyeokKim/MonoDINO-DETR#2]
Hello！Thanks for your nice work，first, I want to visualize the detection result，could you tell me how to do? Second，how to evaluate the result on the test set? I've only seen the val set in the dataset?   

If you can help me to solve the two questions，I would thank you very much!!!

**Comment by JihyeokKim** [comment]
Hi,

Thank you for your interest.

For visualization, I referred to the following code from MonoGround: [MonoGround Visualization Code](https://github.com/cfzd/MonoGround/blob/main/engine/visualize_infer.py).

Since KITTI only provides labels for the training data, you need to upload the test results to the official KITTI website for evaluation.

Thank you.

**Comment by zhaochunyang666** [comment]
how to get the test results
